### DISCUSSION TITLE
lists PRIVATE or PUBLIC?

### DISCUSSION THREAD

**Discussion by MidhiaQueiroz** [community/community#69024] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

So basically, make it so that you toggle if your list(s) are:
Private, meaning only you can see them;
Registered, meaning only users that are logged in can see them and;
Public, meaning everyone can see them!

**Comment by mehulrajdev** [comment, marked as answer]
Great
